controller/Dockerfile

```diff
@@ -116,8 +116,8 @@ ENTRYPOINT ["./retina/initretina"]
 
 
 # agent final image
-# mcr.microsoft.com/cbl-mariner/distroless/minimal:2.0
-# mcr.microsoft.com/cbl-mariner/distroless/minimal@sha256:63a0a70ceaa1320bc6eb98b81106667d43e46b674731ea8d28e4de1b87e0747f
+# For debug: mcr.microsoft.com/cbl-mariner/distroless/debug:2.0
```

Comment:
change this comment back to the original one? since it was identifying which image is being used for the actual agent

Reply:
Don't want to keep updating this comment. I want to add the debug image as it's very useful for debugging purpose during development.